### PR TITLE
Changed logic for unwovenClassesDir + testcase

### PR DESCRIPTION
**Pull request by amihaiemil**
This concerns issue #45 . In the original implementation, ``unwovenClassesDir`` path had to be specified in order for the copy to be triggered. Now the logic was changed due to OP's request: the copy is always done in the folder ``${project.build.directory}/unwoven``, unless the parameter ``<disableCopy>`` is specified ``true`` - in which case the copy is not performed.

### CODE REVIEW

**Comment by amihaiemil** [comment]
@dmarkov Can you please assign this to @mkordas ? He already knows the topic. Thanks.

**Comment by dmarkov** [comment]
@amihaiemil Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@hdouss this pull request is for you, please review

**Comment by amihaiemil** [comment]
@HDouss as initial solution for issue #45, I made PR #46 (basically, the user had to specify ``<unwovenClassesDir>`` in the plugin's configuration, in order to have the unwoven classes copied to that path). 
Then the author of the issue requested the following change: the unwoven classes are always copied under ``${project.build.directory}/unwoven``, and the user can disable this by specifying ``<disableCopy>true</disableCopy>`` in the plugin's configuration - this PR is to solve that.

File: src/it/ajc_disableCopy/pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (HDouss) [inline src/it/ajc_disableCopy/pom.xml]: @amihaiemil Please put the namespaces declarations on different lines.

**Comment by HDouss** [comment]
>the unwoven classes are always copied under ${project.build.directory}/unwoven

@amihaiemil I think that this is the default unwoven target, but we should let this configurable (in case copy is not disabled). I suggest to let an IT with a "custom" `<unwovenClassesDir>`.

**Comment by amihaiemil** [comment]
@HDouss What you said, was the previous solution - it was configurable by specifying any path you wanted. And if you didn't want the copy to happen, you simply wouldn't specify anything. The way it is in this PR is how the author asked, specifically. @krzyk Should I also make ``<unwovenClassesDir>`` customizable?

**Comment by HDouss** [comment]
@amihaiemil The problem with the previous solution was not that `<unwovenClassesDir>` is customizable. The problem was that by default (i.e: you simply wouldn't specify anything) it didn't copy. What is asked is:
1. make it customizable (present in previous solution, absent here)
2. If nothing is mentioned, copy by default to 'unwoven' dir (present here, but not in the previous)

**Comment by krzyk** [comment]
@amihaiemil Exactly as @HDouss wrote, this is the functionality I'm looking for.

**Comment by amihaiemil** [comment]
@HDouss @krzyk Ok, I will make the changes later today. Thanks for clarifications.

**Comment by amihaiemil** [comment]
@HDouss done. Made ``<unwovenClassesDir>`` customizable + test for that case and put namespaces on different lines.

**Comment by HDouss** [comment]
@amihaiemil Thanks !

**Comment by HDouss** [comment]
@rultor merge.

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@hdouss thank you, added **15 mins** to your acc, payment num is `80109503`, 24 hours and 55 mins spent in total to complete

added +15 to your rating, now it is equal to [+3901](http://www.netbout.com/b/36032?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**